### DISCUSSION TITLE
Thank you ❤

### DISCUSSION THREAD

**Discussion by abergs** [polymind-inc/acmebot#242] [General]
I just wanted to say thank you for creating this project and the appservice version. It makes it A LOT easier to work with LE certificates on Azure. I know from experience open source is a lot of work. want to express my gratitude and that I'm very impressed with what you have created, shared openly and the docs you have written to help others get going.

Have a good 2021!
Best wishes,
Anders in Sweden.

**Comment by root-hal9000** [comment]
I will second this. Something that Azure should be just providing in the first place, and you are covering the gap here for us!

**Comment by shibayan** [comment]
I didn't want to close this issue, so I moved it to Discussion 😄

**Comment by booyaa** [comment]
Thank you @shibayan this project has helped me solve the tricky problem of provisioning, storing and renewing HTTPS certs. Really appreciate all the hard work you've put into this.

**Comment by gabador-mof** [comment]
I would add to the praise - Your documentation has helped a lot - it was clear, concise, and got me where other solutions wouldn't. Thank you for creating this project.
